pandas/tests/frame/methods/test_convert_dtypes.py

```diff
@@ -34,6 +34,22 @@ def test_convert_empty(self):
         # Empty DataFrame can pass convert_dtypes, see GH#40393
         empty_df = pd.DataFrame()
         tm.assert_frame_equal(empty_df, empty_df.convert_dtypes())
+    
+    def test_convert_empty_categorical_to_pyarrow(self):
+        df = pd.DataFrame(
+            {
+                "A": pd.Series(pd.Categorical([None] * 5)),
+                "B": pd.Series(pd.Categorical([None] * 5, categories=["B1", "B2"])),
```

Comment:
No need to wrap in `pd.Series` here; supplying `pd.Cateogrical` is sufficient in the constructor.